### PR TITLE
[fix] Added box rendering types, improved named module exporting, and style updates/fixes for text and small bounds

### PR DESCRIPTION
**Pull request by 64blit**
Added a box rendering types of: SimpleSelect, and Simple, for SST

Added export * ./... in the index files, as well as a ` "declaration": true,` to the tsconfig - exporting more of the modules as consumers would expect

Made text render inside the bounds more consistently, adjusting for box height. Added thicker lines to pose also

### CODE REVIEW

File: examples/web/static/multidownload.html [hunk 1]
```diff
@@ -0,0 +1,583 @@
...
+                <div class="col text-center" id="list-models">
+                    <h1 class="display-4">Model Viewer</h1>
+                    <div id="list-models-result"></div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <script>
+
+        const apiKey = 'AAGcsWj8N2PlKQl9c9ydz3QFZ0FBQUFBQm1mZDB5eDUwalNlYi12NWotd3hsVGJiMW1sVXF1dE9aOU9oSGVBOWtBQXoxZmNjUE5Nb1YzY3RROUdzbVUwUkZtcDhZcG5vSWROTzR1TU8ybGhZckx6RTgzYVZwMjZEREZjalZubnpYaUNMWVdBODg9';
```
Comment (tschulz) [inline examples/web/static/multidownload.html:43]: Remove this secret from source code.

Now that you have committed it, this secret is compromised and has to be invalidated asap. Please confirm.

File: examples/web/static/multidownload.html [hunk 2]
```diff
@@ -0,0 +1,583 @@
...
+        const auth = apiKey ? { secretKey: apiKey } : { oAuth2: true };
+       
+        const account_uuid = "49326f2e085a46c39ba73f91c52e436c";
```
Comment (tschulz) [inline examples/web/static/multidownload.html:46]: treat account ids as weak secrets too. They should not be exposed if not needed.

File: src/eyepop/semaphore.ts [hunk 3]
```diff
@@ -10,7 +10,8 @@
  * by functions that wait for permits to become available. This makes it possible
  * to use async/await to synchronize your code.
  */
-export class Semaphore {
+export class Semaphore
+{
```
Comment (tschulz) [inline src/eyepop/semaphore.ts:14]: I don't like hanging curly - seems like wasting a line for nothing

**Review by tschulz — CHANGES_REQUESTED** [review]
pls revert those formatting changes with the extra line for blocks

File: examples/web/static/multidownload.html (hunk 1 above)
Comment (64blit) [reply examples/web/static/multidownload.html:43]: Whoops, didn't catch that. I'll let Andy know right now to generate a new key.

**Comment by 64blit** [comment]
Fixes pushed, does that cover everything? @tschulz 

Also added the prettier formatter in another branch, will make a PR once this one is gtg

**Review by tschulz — APPROVED** [review]
lgtm

**Pull request merged**